### ISSUE TITLE
Write a scope for phrosty

### ISSUE THREAD

**Issue by wmwv** [laldoroty/phrosty#14]
It might be helpful to write down a scope document to figure out what should live in phrosty and what should live in diff-img or should be its own utils?

**Comment by laldoroty** [comment]
Agree. I think there is now a lot of stuff that should live in phrosty that's in diff-img...
